Makefile

```diff
@@ -160,6 +160,11 @@ build-docker: require-APP build-docker-$(APP)
 build-docker-%: nix
 	@nix-build nix/docker.nix -A $(subst build-docker-,,$@) -o result-docker-$(subst build-docker-,,$@) --fallback
 
+test: require-APP
```

Comment:
This is temporary: it should be triggered in build-app by Nix.
@garbas any input on that ?